ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host1/underlay_ospf_authentication.yml

```diff
@@ -0,0 +1,8 @@
+---
+underlay_routing_protocol: ospf
+underlay_ospf_authentication:
```

Comment:
Show we add tests for deprecated key as part of this PR?

Reply:
yes - this is what is done here

Reply:
cf comment line 7